x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments/assignments.cy.ts

```diff
@@ -77,42 +67,23 @@ describe('Alert user assignment - ESS & Serverless', { tags: ['@ess', '@serverle
     });
 
     it('alert with some assignees in alerts table', () => {
-      const users = [ROLES.detections_admin, ROLES.t1_analyst];
+      const users = [getDefaultUserName()];
       updateAssigneesForFirstAlert(users);
       alertsTableShowsAssigneesForAlert(users);
     });
 
     it(`alert with some assignees in alert's details flyout`, () => {
-      const users = [ROLES.detections_admin, ROLES.t1_analyst];
+      const users = [getDefaultUserName()];
       updateAssigneesForFirstAlert(users);
       expandFirstAlert();
       alertDetailsFlyoutShowsAssignees(users);
```

Comment:
Nit: it would be nice to have some convention (maybe either function naming (`assert` as a prefix), or folder location (`/assertions`), or both) to identify these tasks as performing assertions.

I know that for a while @MadameSheema was requesting that we not abstract assertions into helpers at all, but I think if we do so we should make those assertions a bit more discoverable.

Reply:
@MadameSheema any thought/preferences on this one?

Reply:
Hey!! the overall preference is to NOT abstract assertions.